.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,15 +19,15 @@ Here are some guidelines to help the review process go smoothly.
    noted here: https://help.github.com/articles/closing-issues-using-keywords/
 
 5. If your pull request is not ready for review but you want to make use of the
-   continuous integration testing facilities please label it with `[WIP]`.
+   continuous integration testing facilities please mark your pull request as Draft.
```

Comment:
```suggestion
   continuous integration testing facilities please [mark your pull request as Draft](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request#converting-a-pull-request-to-a-draft).
```

Reply:
So even though this is a markdown `.md` file the text here does not get rendered as markdown because of the `<!-- -->` at the beginning/end. Here is a picture of what it looks like for me in the browser (I have dark-mode theme enabled).

![pull-request1](https://user-images.githubusercontent.com/45795991/113314221-13581e80-92da-11eb-9eeb-39262e320a0f.png)

And the Preview panel does not show anything here.

![pull-request2](https://user-images.githubusercontent.com/45795991/113314272-27038500-92da-11eb-9bbd-d59d2bb8434e.png)

The links are helpful but I think I will include them without the markdown formatting.

Reply:
Yes I forgot this was all inside a comment. Good idea to do it without markdown.